### PR TITLE
Clarify creating/configuring client

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

File: docs/learning/tutorials/develop-desktop-viewer.md [hunk 1]
```diff
@@ -5,15 +5,33 @@
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
 - [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
 
+## Create client
+
+- Navigate to [developer.bentley.com](https://developer.bentley.com/register/)
```
Comment (pmconne) [inline docs/learning/tutorials/develop-desktop-viewer.md:10]: The term "client" doesn't appear anywhere on the linked page. Are you using it as a synonym for "application"?

Comment (williamkbentley) [reply docs/learning/tutorials/develop-desktop-viewer.md:10]: Hmm yes. I do seem to be mixing terms between client and application. I'll settle on Application for this portion.

**Review by pmconne — APPROVED** [review]

File: docs/learning/tutorials/develop-desktop-viewer.md [hunk 2]
```diff
@@ -5,15 +5,33 @@
...
+- Sign in
+- Configure client
+  - Provide client name
+  - API associations: "Visualization"
+  - Application type: "Desktop/Mobile"
+  - Redirect URI: "http://localhost:3000/signin-callback"
+  - Post logout redirect URI: "http://localhost:3000"
+  - Save
+- View Details
```
Comment (Josh-Schifter) [inline docs/learning/tutorials/develop-desktop-viewer.md]: I suggest we should be more explicit in these two steps.

File: docs/learning/tutorials/develop-desktop-viewer.md [hunk 3]
```diff
@@ -5,15 +5,33 @@
...
+- Copy Client ID
+
 ## Build
 
 - Open the cloned repo in VS Code
+- Edit [./src/backend/main.ts](https://github.com/imodeljs/desktop-starter/blob/master/src/backend/main.ts#L55)
+- Replace `process.env.ELECTRON_CLIENT_ID` with your Client ID
```
Comment (Josh-Schifter) [inline docs/learning/tutorials/develop-desktop-viewer.md:26]: I suggest adding an example of what the line looks like after being edited.

**Pull request merged**